client/view/Form/form.js

```diff
@@ -17,8 +18,8 @@ const errorMessage = document.querySelector('[data-id=error_message]')
 
 form.addEventListener('submit', handleFormSubmit)
 
-textarea.addEventListener('keypress', e => {
-  if (e.key === 'Enter' && !e.shiftKey) {
+textarea.addEventListener('keydown', e => {
+  if (!e.shiftKey && e.key === 'Enter') {
```

Comment:
> Update hotkeys.Enter for sumbit, Shift+Enter for newline

Mobile phones have not Shift+Enter. Let’s better:
1. Send queries without any Enter just after changes with some debounce
2. `Enter` adds new line
3. `Shift+Enter` also send request

Reply:
Done. It's the best solution.